bin/cml/runner.js

```diff
@@ -432,10 +432,12 @@ exports.builder = (yargs) =>
           'One or more user-defined labels for this runner (delimited with commas)'
       },
       idleTimeout: {
-        type: 'number',
-        default: 5 * 60,
+        type: 'string',
+        default: '5 minutes',
+        coerce: (val) =>
+          /^-?\d+$/.test(val) ? parseInt(val) : timestring(val),
         description:
-          'Seconds to wait for jobs before shutting down. Set to -1 to disable timeout'
+          'Time to wait for jobs before shutting down. Set to -1 to disable timeout'
```

Comment:
`-1` or `never` 😉

Reply:
I was thinking of adding an `infinity` / `never` / `disabled` / whatever keyword, but nested ternary conditionals trigger the OCD I don't have. 😉 

Jokes apart, it's a great idea. Let's do it!

Reply:
```suggestion
          'Time to wait for jobs before shutting down. Set to -1 or "never" to disable timeout'
```

Reply:
```suggestion
          'Time to wait for jobs before shutting down (e.g. "5min"). Use "never" to disable'
```